draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -580,9 +580,20 @@ for their contributions (some of which substantial) to this draft and to the ini
 
 ### New Entries to the COSE Header Parameters Registry
 
-This document requests IANA to add new values to the 'COSE Algorithms' and to the 'COSE Header Algorithm Parameters' registries in the 'Standards Action With Expert Review' category.
+This document requests IANA to add new values to the 'COSE Header Parameters' registries in the 'Integer values from 256 to 65535' range with 'Specification Required' Registration Procedure. This requests includes an earmarked label range from 350-399 and three specific labels in that earmarked range elaborated on below.
 
-#### COSE Header Algorithm Parameters
+#### COSE Header Parameters
+
+##### COSE Receipts Earmark Range
+
+This request to IANA includes a range of labels for COSE header parameters to be "Earmarked for COSE Receipts" so that numbers in that earmark range ca be reassigned by IANA based on advice from the designated experts for the COSE Header Parameters registry.
```

Comment:
```suggestion
This request to IANA includes a range of labels for COSE header parameters to be "Earmarked for COSE Receipts" so that numbers in that earmark range can be reassigned by IANA based on advice from the designated experts for the COSE Header Parameters registry.
```